TextMining.py

```diff
@@ -0,0 +1,125 @@
+from bs4 import BeautifulSoup
+import requests
+html = BeautifulSoup(requests.get('https://en.wikipedia.org/wiki/Franklin_W._Olin_College_of_Engineering').text, 'lxml')
+html.find('p')  # find the first paragraph
+str(html.find('p'))  # the first paragraph, as a string. Includes embedded <b> etc.
+import re
+str(re.sub(r'<.+?>', '', str(html.find('p'))))
+alice_in_wonderland_full_text = requests.get('https://www.gutenberg.org/files/11/11-0.txt').text
+alices_adventures = requests.get('http://www.gutenberg.org/cache/epub/19002/pg19002.txt').text
```

Comment:
It is probably good idea to separate a data fetching functionality and data analysis functionality into different python files. This way, you don't have to request data every time you run an analysis.